### ISSUE TITLE
Change inactive page message to allow resending activation link

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/topcoder-app#909]
When an inactive user tries to log in, they should be able to trigger an email with their activation link

**Comment by fnisen** [comment]
Can I be involved in this one please? I think I originally created this issue.

**Comment by talesforce** [comment]
@fnisen - Do you have any specific inputs on this? I was hoping to include this in the next challenge that I launch. Should I hold off until I hear from you on this? cc @ajefts

**Comment by fnisen** [comment]
@talesforce yes, here is the scenario I'd like to support. Right now when a user registers for Topcoder (both member and customer), they need to confirm their email address by clicking an activation link that we send them. If they try to log in before confirming their email address, they'll see this page/message: 

<img width="1276" alt="screen shot 2016-12-28 at 12 34 59 pm" src="https://cloud.githubusercontent.com/assets/12552247/21531078/24f85bd2-ccfa-11e6-8eb2-e43ed4f65768.png">

This is also the same page shown right after the user first registers, which is confusing. So, what'd like is: 

1. One page/message for the scenario of the user just having completed registration and our telling them to go check their email (can keep the page shown in the screen shot)
2. Create a new page/message for the scenario of the user having registered some time ago, but still not having activated their account. 

On both pages, they should be able to trigger the activation link again (without needing to contact support). We also need some thinking on how this should work: 
- should they get the option to trigger the link regardless of how much time has elapsed since registration or should we cap it? (e.g., only can self-trigger for 6 months, but then need to contact support; what are the pros, cons, risks?)
- should there be a limit on how often the email can be triggered, so someone doesn't flood our email service?
- other issues that may come up if we expose self-triggering?

**Comment by birdofpreyru** [comment]
Any update on this?

**Comment by fnisen** [comment]
@birdofpreyru no, no work has been done since my comment, as far as I know. You're welcome to pick it up. It may be too large of a scope for a bug bash, so you're welcome to run a challenge or whatever you think is needed.
